src/lib/react/__tests__/useLocalMedia.unit.ts

```diff
@@ -0,0 +1,60 @@
+import { jest } from "@jest/globals";
```

Comment:
is the `.unit` naming convention something new? It's the first time I'm seeing it. Usually we would suffix test files with `.spec`.

Reply:
I might just have seen this in some other project of ours, but happy to use spec 👍

Reply:
Renamed